wiki/general/tutorials/getting-started-.md

```diff
@@ -0,0 +1,120 @@
+In that tutorial we will cover how to use Launchpad to generate a WordPress plugin.
+
+### Create the project
+
+#### Generate the project using composer
+The first step in creating the project itself is by using the composer command line with the following input: `composer create-project wp-launchpad/launchpad my-plugin`
+
+Once this is completed, you should be able to see a new folder with the code for your project inside it. However, it is not yet ready to be used as it is still Launchpad default code and needs to be initialized.
+
+#### Initialize project with launchpad command
+
+As multiple projects can exist on the same WordPress website at once, it is important to containerize all of them, and for that purpose we will use a specific namespace for our code.
+
+To do so with Launchpad, we need to run the following command in the terminal:
+`bin/generator initialize`
+
+Once the command has been executed, it should ask you for your plugin name, and it will generate the necessary changes from it.
+
+You can assert the command executed successfully by checking if the name from your plugin file matches what was the name from your plugin.
+
+### Install a module in your project using composer
+
+Now that the project is set up properly, it is now time to install some modules to gain time and effort when working on your project.
+
+A good option would be to use the WordPress options facade to inside the module `wp-launchpad/framework-options`.
```

Comment:
A part is missing in this sentence? It would be good to explain quickly what this module does.